packages/react-native-web/src/exports/Image/index.js

```diff
@@ -236,10 +236,10 @@ const Image = forwardRef<ImageProps, *>((props, ref) => {
         function load(e) {
           updateState(LOADED);
           if (onLoad) {
-            onLoad();
+            onLoad(e);
```

Comment:
This should be `{ nativeEvent: e }` to mirror React Native

Reply:
If I understood correctly, this `onLoad` use the `ImageLoader.load`, so, if I change it there to `{ nativeEvent: e }`, here I can't recreate the event because it will be duplicated like `{ nativeEvent: { nativeEvent: e } }`